CHANGELOG.md

```diff
@@ -891,13 +917,11 @@ class Node:
 
 
 @strawberry.type
-class Video(Node):
-    ...
+class Video(Node): ...
```

Comment:
Although black and ruff are formatting those like this lately, I personally think that for docs they look less readable 🤔

Reply:
same

Reply:
doesn't look like it is possible :'D